Squeezer/src/main/java/uk/org/ngo/squeezer/framework/BaseActivity.java

```diff
@@ -120,22 +121,28 @@ public void onServiceDisconnected(ComponentName name) {
         }
     };
 
+    protected boolean addActionBar(){
+        return true;
+    }
+
     @Override
     @CallSuper
     protected void onCreate(android.os.Bundle savedInstanceState) {
         mTheme.onCreate(this);
         super.onCreate(savedInstanceState);
 
-        // Set the icon as the home button, and display it.
-        ActionBar actionBar = getSupportActionBar();
-        if (actionBar != null) {
-            actionBar.setHomeAsUpIndicator(R.drawable.ic_action_home);
-            actionBar.setDisplayHomeAsUpEnabled(true);
+        if (addActionBar()) {
+            // Set the icon as the home button, and display it.
+            ActionBar actionBar = getSupportActionBar();
+            if (actionBar != null) {
+                actionBar.setHomeAsUpIndicator(R.drawable.ic_action_home);
+                actionBar.setDisplayHomeAsUpEnabled(true);
+            }
         }
 
         boundService = bindService(new Intent(this, SqueezeService.class), serviceConnection,
                 Context.BIND_AUTO_CREATE);
-        Log.d(getTag(), "did bindService; serviceStub = " + getService());
+        Log.d(TAG, "did bindService; serviceStub = " + getService());
```

Comment:
I thought this was confusing because it reports the subclass in the log message but the code is defined in the super class; I'm unfamiliar with the 'best practice' or 'most pervasive way this is done' with android so I would be fine changing it back; but I introduced a lot of static final TAGs as part of the logging code I added which should probably be consistent with whatever choice is made.

Reply:
I'm good with this. I'll clean up the getTag stuff when this is fully merged.